<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountEditWindow</class>
 <widget class="QWidget" name="AccountEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 255, 252);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>add Card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editCardButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>edit Cards</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewCardsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>view Cards</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid red;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid blue;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;customer account options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
